src/applications/vaos/utils/error.js

```diff
@@ -21,4 +22,9 @@ export function captureError(err, skipRecordEvent) {
   if (!skipRecordEvent) {
     recordVaosError(eventErrorKey);
   }
+
+  if (!environment.isProduction()) {
+    // eslint-disable-next-line no-console
+    console.error(err);
```

Comment:
Because we wrap errors in this function and log them to sentry, they're not printed out as usual in the console, which makes debugging harder.